extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -218,22 +218,22 @@ const OHIFCornerstoneViewport = React.memo(props => {
     }
 
     const loadViewportData = async () => {
-      const viewportData = await CornerstoneCacheService.getViewportData(
+      await CornerstoneCacheService.getViewportData(
         viewportIndex,
         displaySets,
         viewportOptions.viewportType,
         dataSource,
+        (viewportDataLoaded) => {
```

Comment:
Pass in the necessary use of the viewport data to allow the event to fire after the data is stored/used